src/calendar/day/dot/index.tsx

```diff
@@ -1,3 +1,4 @@
+import PropTypes from 'prop-types';
```

Comment:
Why are we using prop-types here?

Reply:
`marking` is using Dot props. We should include Marking component in this PR...

Reply:
The `extractComponentProps` function (src/component-updater.js) is based on the prop-types, when removing it the unit test are failling. So we decided to keep it until we'll make the relevant adjustment..

Reply:
Ok